hkpy/hklib/node.py

```diff
@@ -139,6 +142,66 @@ def to_dict(self) -> Dict:
 
         return jobj
 
+
+class HKDataNode(HKAnyNode):  
+    """ 
```

Comment:
Would be nice to explain what is an HKDataNode here

Reply:
done